### ISSUE TITLE
Bug report: Fix for Qt-Linguist cursor browsing

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#11934]
Qt-Linguist is a localization translation tool for the qt gui
During the operation of Qt-Linguist with NVDA the cursor cannot navigate through the translation entries in the chart and the items in the list.
### Steps to reproduce:
First open download [Qt-Linguist5.15.2](https://github.com/thurask/Qt-Linguist/releases/download/20201205/linguist_5.15.2.zip)
Then unzip linguist_5.15.2.zip and use linguist.exe to add in the .ts translation file to reproduce the error.
### Actual behavior:
After opening linguist.exe and loading the translation file NVDA cannot use the cursor to browse the interface of linguist.exe
#### NVDA installed/portable/running from source:
install and portable
#### NVDA version:
alpha-21505
#### Windows version:
Microsoft Windows [version 10.0.19042.685]
#### Name and version of other software in use when reproducing the issue:
linguist_5.15.2
#### Other information about your system:
none
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
none
#### If addons are disabled, is your problem still occuring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
This issue is not managed by this tool
log:
[linguist_5.15.2.log](https://github.com/nvaccess/nvda/files/5682642/linguist_5.15.2.log)
thanks

**Comment by Adriani90** [comment]
@dpy013 are you still having this issue in Qt-Linguist 6.7.1 / or 5.15.13?

**Comment by Adriani90** [comment]
https://github.com/thurask/Qt-Linguist/releases

**Comment by dpy013** [comment]
hi @Adriani90 
Haven't tested it yet, I'll check it out sometime.

**Comment by dpy013** [comment]
> @dpy013 are you still having this issue in Qt-Linguist 6.7.1 / or 5.15.13?

Just tested linguist_6.7.1 with NVDA 2024.2 Portable and the problem is still not solved.

**Comment by Adriani90** [comment]
@dpy013 could you please also open a new issue in their reporsitory? It is here:
https://github.com/thurask/Qt-Linguist/releases

You can post the link here so we can track the issue later on.
